server/configmodule.py

```diff
@@ -104,7 +104,7 @@ class LocalPrivateConfig(PrivateConfig):
     AI_CONFIG_PATH = os.path.abspath(
         os.path.join(os.path.curdir, '..', 'deploy/overlays/local/ai.yaml'))
     LOCAL = True
-    SECRET_PROJECT = 'datcom-website-private'
+    SECRET_PROJECT = 'datcom-feedingamerica'
```

Comment:
Can you create new classes like LocalIitmConfig? Can check what is changed in this PR https://github.com/datacommonsorg/website/pull/1497/files

Reply:
It already exists for feeding america. Do they need to be modified?